### ISSUE TITLE
17.9 Preview, task bars etc...

### ISSUE THREAD

**Issue by djwilson7** [T0uchM3/Cyberpunk-Theme#21]
Hey, on the new preview of 17.9 2022, there seems to be some conflict with the outer edges showing as white instead of the dark blue background, Pic included shows preview 17.9 up top, and 17.8 below. My best assumption, there is some conflict with the new tint feature they plan on rolling out.
![Screenshot 2023-11-21 115227](https://github.com/T0uchM3/Cyberpunk-Theme/assets/145996575/32968aa5-4de5-4016-8c1b-96249ff57cbf)

**Comment by h4zm1** [comment]
Yeah that's happening cause the new ui option for vs 22 is enabled I assume, which is still in preview.
If you read this article https://devblogs.microsoft.com/visualstudio/the-visual-studio-ui-refresh-preview-is-here down below they will mention that there are compatibility issues with 3rd party themes and they are working on it.

**Comment by mask06** [comment]
The colors of warnings and error messages within the theme are very similar, making it difficult to distinguish between them; they should be changed."
